src/java/edu/ucsd/library/dams/triple/TripleStoreUtil.java

```diff
@@ -320,6 +335,49 @@ private static void loadRDF( InputStream in, boolean deleteFirst,
 		{
 			throw new TripleStoreException( "Error processing triples", ex );
 		}
+
+		return null;
+	}
+	public static Set<String> validateModel( Model model, Set<String> validClasses,
```

Comment:
Would you prefer this merged then for now, and we revisit if/when we get additional validation requirements?